packages/schema/src/types/config.ts

```diff
@@ -4,50 +4,16 @@ import type { Options as VuePluginOptions } from '@vitejs/plugin-vue'
 import type { Options as VueJsxPluginOptions } from '@vitejs/plugin-vue-jsx'
 import type { SchemaDefinition } from 'untyped'
 import type { NitroRuntimeConfig, NitroRuntimeConfigApp } from 'nitropack'
+// TODO: expose SnakeCase type from scule
```

Comment:
```suggestion
```

Reply:
@danielroe Would https://github.com/unjs/scule/pull/58 need to be merged and released first?

Reply:
Yes, it will.